### ISSUE TITLE
Documentation catch-all

### ISSUE THREAD

**Issue by raoulmillais** [paritytech/substrate-connect#456]
This is a ticket to capture decisions we make about expectations we have of the app developers so that we can write it up before releasing.

* If a user opens the same app in 2 tabs, the second tab will fail to connect to the extension and the app will receive and error.  They should handle this and show in their UI that they failed to connect.

**Comment by tomaka** [comment]
> If a user opens the same app in 2 tabs, the second tab will fail to connect to the extension and the app will receive and error. They should handle this and show in their UI that they failed to connect.

Wait, but why?

**Comment by raoulmillais** [comment]
Yeah it doesn't have to be this way and it's super easy to fix :+1:  We do still need to document the error cases and what we expect the app developer to do though.

**Comment by raoulmillais** [comment]
Fixed in https://github.com/paritytech/substrate-connect/pull/459

**Comment by sacha-l** [comment]
Pasting some instructions that @wirednkod wrote to address #465 : 

When you run the `yarn dev:extension` command (chrome version) you need to add the extension manually to your browser.
You can follow the steps:

1. Write on your address bar: chrome://extensions/
2. Make sure the Developer Mode is activated (top right)
3. Click Load unpacked (top left)
4. Navigate and open the directory substrate-connect-repo/projects/extension/dist
5. The extension should be now installed.

--> I think this will help clarify how to launch the extension. IIRC the README also indicates that a browser window opens after running the `yarn dev:extension` command which I don't believe is true... for now ? 😅

**Comment by raoulmillais** [comment]
The docs should include instructions for Firefox too

**Comment by wirednkod** [comment]
This is covered by last bullet of  #572. No reason to still be open
